### DISCUSSION TITLE
Elastalert stopped working - Error: No such container: so-elastalert - After 2.4.70 update

### DISCUSSION THREAD

**Discussion by jrm360seclab** [Security-Onion-Solutions/securityonion#13168] [General]
Good day all, for some reason I am noticing that after the update now I can't access Kibana, and ElastAlert is giving the error above, any help???

**Comment by dougburks** [comment]
Did soup complete successfully or were there errors?

What version were you running before the update?

What are the hardware specs of your system?

How is it configured?

Have you checked the Elastalert logs?
https://docs.securityonion.net/en/2.4/elastalert.html#diagnostic-logging

Have you checked the Kibana logs?
https://docs.securityonion.net/en/2.4/kibana.html#diagnostic-logging

Have you checked the Elasticsearch logs?
https://docs.securityonion.net/en/2.4/elasticsearch.html#diagnostic-logging
